packages/ocs/dashboards/persistent-external/fallback-hook.ts

```diff
@@ -0,0 +1,37 @@
+import { StorageClassModel } from '@odf/shared/models';
+import { StorageClassResourceKind } from '@odf/shared/types';
+import { referenceForModel } from '@odf/shared/utils';
+import {
+  WatchK8sResource,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+const scResource: WatchK8sResource = {
+  isList: true,
+  kind: referenceForModel(StorageClassModel),
+};
+
+const fsProvisionerPostFix = '.cephfs.csi.ceph.com';
+
+/**
+ * This hooks is for ODF Client Mode use case only.
+ * ODF client mode doesn't deploy ODF StorageCluster hence falling back to StorageClases.
+ */
+const useFallbackHook = (storageClusterName) => {
+  const [storageClasses, storageClassesLoaded, storageClassLoadError] =
+    useK8sWatchResource<StorageClassResourceKind[]>(scResource);
```

Comment:
this way it won't even poll when we already have `storageClusterName`...
```suggestion
   useK8sWatchResource<StorageClassResourceKind[]>(getValidWatchK8sResourceObj(scResource, !storageClusterName));
```

Reply:
this resource is already called in the same page for inventory card. so we can live with this.

Reply:
make sense then !! (I did not check the parent component)